### ISSUE TITLE
settings module does not work in safari incognito

### ISSUE THREAD

**Issue by lorenzhs** [glowing-bear/glowing-bear#591]
```
Error: QuotaExceededError: DOM Exception 22
setItem@[native code]
set@https://latest.glowing-bear.org/pull/590/js/localstorage.js:57:28
set@https://latest.glowing-bear.org/pull/590/js/settings.js:20:15
setDefaults@https://latest.glowing-bear.org/pull/590/js/settings.js:63:9
https://latest.glowing-bear.org/pull/590/js/glowingbear.js:21:25
```

**Comment by lorenzhs** [comment]
Labelled as iOS but also affects Mac OS

**Comment by lorenzhs** [comment]
Fixed in #971
